streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedStoreTest.java

```diff
@@ -130,6 +134,8 @@ public void shouldMigrateDataFromDefaultToTimestampColumnFamily() throws Excepti
         // approx: 7 entries on old CF, 0 in new CF
         assertThat(rocksDBStore.approximateNumEntries(), is(7L));
 
+        // prefix scan should return 7 keys with prefix "key"
+        assertThat(runPrefixScan("key"), is(7));
```

Comment:
I guess, it would be more consistent to add your verifications to `iteratorsShouldNotMigrateData()` where all other iterators are tested.